app/models/main.py

```diff
@@ -1653,13 +1656,37 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
         if errs:
             raise ValidationError(errs)
 
+    class query_class(BaseQuery):
+        def has_datetime_field_between(self, attr, start_datetime, end_datetime, inclusive=None):
+            """
+            Date filter values should be datetime objects. e.g.
+                BriefResponse.query.has_date_field_between(
+                    'published_at', datetime(2017, 1, 1), datetime(2017, 1, 2, 23, 59, 59, 999999), inclusive=True
+                )
+            would return brief responses published between 2017-01-01T00:00:00.000000Z and 2017-01-02T23:59:59.999999Z.
+            """
+            if not (isinstance(start_datetime, datetime) and isinstance(end_datetime, datetime)):
+                raise ValueError('Datetime object required')
+            if inclusive:
+                return self.filter(
+                    sql_and(
+                        getattr(BriefResponse, attr) >= str(start_datetime),
+                        getattr(BriefResponse, attr) <= str(end_datetime)
+                    )
+                )
+            return self.filter(
+                sql_and(
+                    getattr(BriefResponse, attr) > str(start_datetime),
+                    getattr(BriefResponse, attr) < str(end_datetime)
+                )
+            )
+
```

Comment:
This feels really unnecessary and special-casey to me. I mean - it's just a range test, two comparisons - why make the way we handle range tests be special an different for dates on `BriefResponse`s?

Reply:
While it's true that our models do have some of these weird special-casey comparison methods in their `query_class`es, but they were generally written a long time ago and it's not a pattern I think we want to continue following.

Reply:
oh look we said the same thing, sorry, not intending to pile on, but FWIW I agree with this